cmd/internal/flowmode/cmd_run.go

```diff
@@ -49,15 +52,18 @@ func runCommand() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "run [flags] file",
+		Use:   "run [flags] path",
 		Short: "Run Grafana Agent Flow",
 		Long: `The run subcommand runs Grafana Agent Flow in the foreground until an interrupt
 is received.
 
-run must be provided an argument pointing at the River file to use. If the
-River file wasn't specified, can't be loaded, or contains errors, run will exit
+run must be provided an argument pointing at the River path to use. If the
```

Comment:
somewhere in here I think we should explicitly say we also accept a filepath. This through me off for a minute until I read the code.